src/main/views/footer.njk

```diff
@@ -0,0 +1,40 @@
+{% from "govuk/components/footer/macro.njk" import govukFooter %}
```

Comment:
Why the need to separate footer.njk? You're extending template from the new cya.njk anyway.

Reply:
Im using separate footer.njk because in some places i need to override it without contacts in the bottom(to avoid code duplication)